File: yuqi/yuqi-sellergoods-interface/src/main/java/com/yuqi/sellergoods/service/OrderService.java
package com.yuqi.sellergoods.service;

import com.yuqi.pojo.TbOrder;
import entity.PageResult;
import entity.Result;
import group.Order;

import java.util.List;

/**
* 服务层接口
* @author Administrator
*
*/
public interface OrderService {

/**
* 返回全部列表
* @return
*/
public List<Order> findAll(String name);


/**
* 返回分页列表
* @return
*/
public PageResult findPage(int pageNum, int pageSize, String name);


/**
* 分页
* @param pageNum 当前页 码
* @param pageSize 每页记录数
* @return
*/
public PageResult findPage(TbOrder order, int pageNum, int pageSize);



    /**
     * 根据ID获取实体
     * @param id
     * @return
     */
    public TbOrder findOne(Long id);





    //    更新订单状态
    public void updateStatus(Long id,String status);
}

File: yuqi/yuqi-shop-web/src/main/webapp/js/service/orderService.js
//服务层
app.service('orderService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../order/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows,loginName){
		return $http.get('../order/findPage.do?page='+page+'&rows='+rows+"&name="+loginName);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../order/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../order/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../order/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../order/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../order/search.do?page='+page+"&rows="+rows, searchEntity);
	}

	this.showName=function(){
		return $http.get('../login/name.do');
	}
	this.updateStatus=function(id,status){
        return $http.get('../order/updateStatus.do?id='+id+"&status="+status);
	}
});
